x-pack/plugins/alerting/server/saved_objects/mappings.ts

```diff
@@ -41,6 +41,7 @@ export const alertMappings: SavedObjectsTypeMappingDefinition = {
       type: 'keyword',
     },
     actions: {
+      dynamic: false,
```

Comment:
Let's check with the Kibana core team, but looking at https://github.com/elastic/kibana/pull/149102, it feels like there's new guidelines where it's better to not set mappings on fields instead of setting `enabled: false` or `index: false`. I guess it allows additive changes in the future instead of trying to update an enabled: false or index: false field?

Reply:
I used `dynamic: false` to discuss here :) 
This is coming from the below discussion.

https://github.com/elastic/kibana/pull/146183/files/f15a6cb429cc41149d31bffee78bfcf3087f78fd#diff-356f716cd6d9da3641f1743fe25aafbbfd94ddf401a8c35a3521b7b2f2c2df87

AFAIU: As long we don't query by the field and we do the validation on API level, there is no need to increase the mappings...

Reply:
All that makes sense. 

I'll miss the alignment with the TS typing, and am sure some day I'll see a "field" in an SO JSON I log out, look at the mapping, see the "field" isn't in the mapping, and freak out :-).  And as mentioned, since ES won't do any validation on these, puts more emphasis on us doing the validation in Kibana - everything looks good with this PR in that regard, I guess I'm just missing the extra added protection :-)